app/api/sendEmail/route.ts

```diff
@@ -39,6 +41,8 @@ export async function POST(request: NextRequest) {
     const email = [
       `From: ${EMAIL_USER}`,
       `To: ${to}`,
+      cc ? `Cc: ${cc}` : '',
+      bcc ? `Bcc: ${bcc}` : '',
       `Subject: ${subject}`,
       '',
       text,
```

Comment:
This doesn't correctly set the email Subject (if you look at the emails sent the subject is in the body)
![image](https://github.com/user-attachments/assets/28578eab-04de-493e-96e8-dc2342e45a13)